os/include/tinyara/wifi/rtk/wifi_constants.h

```diff
@@ -56,6 +56,10 @@ extern "C" {
 
 #define MCSSET_LEN			16
 
+

```

Comment:
Let's add this file in bsp change.

Reply:
You want it to be separated from this PR?

Reply:
No. Let's add it in `[2/5]os/board: Add support for Rtl8720e board`.